spring-petclinic-api-gateway/src/main/resources/application.yml

```diff
@@ -4,6 +4,8 @@ spring:
   config:
     import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/}
   cloud:
+    loadbalancer:
+      configurations: health-check
```

Comment:
**question**: did you test this `health-check` confituration? and how? Did you have some logs?

Reply:
@MarianConstantinMarica  Did you have any logs?